frontend/src/components/config/Theme.tsx

```diff
@@ -5,6 +5,7 @@ export const ThemeConfig: any = {
   colors: {
     text: '#444',
     background: '#f0f4f8',
+    border: '#ccc',
```

Comment:
nit, personal preference (if you don't want to do this, that's fine): Name this as a shade of the color, or as a more specific border name (like `gameBorder`).